TTLocalizer: Running in language: English
OTPLocalizer: Running in language: English
Preloading suits...
Preloading Toon heads...
Preloading avatars...
{401000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x160484E0>}
entered wait for zone redirect
entered wait for set zone response - 2
Reading DNA file... phase_4/dna/storage.pdna
Reading DNA file... phase_3.5/dna/storage_interior.pdna
Reading DNA file... phase_4/dna/storage_TT.pdna
Reading DNA file... phase_4/dna/storage_TT_sz.pdna
Reading DNA file... phase_4/dna/toontown_central_sz.pdna
__handleAvatarChooserDone: 100000194, 'Duck', ('fls', 'ms', 's', 'm', (0.99, 0.69, 0.51, 1.0), (1.0, 1.0, 1.0, 1.0), (0.99, 0.69, 0.51, 1.0), (0.99, 0.69, 0.51, 1.0), 4, 1, 4, 1, 6, 27), 1
entered wait for zone redirect
entered wait for set zone response - 2
Reading DNA file... phase_4/dna/storage.pdna
Reading DNA file... phase_3.5/dna/storage_interior.pdna
Reading DNA file... phase_4/dna/storage_TT.pdna
Reading DNA file... phase_4/dna/storage_TT_sz.pdna
Reading DNA file... phase_4/dna/toontown_central_sz.pdna
Traceback (most recent call last):
  File "C:\Panda3D-1.10.0\direct\distributed\ConnectionRepository.py", line 613, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.10.0\direct\distributed\ConnectionRepository.py", line 620, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 1224, in handleDatagram
    self.handler(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1152, in handleMessageType
    self.handleGenerateWithRequired(di)
  File "otp\distributed\OTPClientRepository.py", line 1282, in handleGenerateWithRequired
    return self.__doGenerate(doId, parentId, zoneId, classId, di, other)
  File "otp\distributed\OTPClientRepository.py", line 1346, in __doGenerate
    self.generateWithRequiredFields(dclass, doId, di, parentId, zoneId)
  File "C:\Panda3D-1.10.0\direct\distributed\ClientRepositoryBase.py", line 254, in generateWithRequiredFields
    distObj = classDef(self)
  File "toontown\ai\NewsManager.py", line 16, in __init__
    base.localAvatar.inventory.setInvasionCreditMultiplier(1)
AttributeError: ToonBase instance has no attribute 'localAvatar'
Traceback (most recent call last):
  File "C:\Panda3D-1.10.0\direct\showbase\ShowBase.py", line 1682, in updateManagers
    dt = globalClock.getDt()
  File "C:\Panda3D-1.10.0\direct\distributed\ConnectionRepository.py", line 613, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.10.0\direct\distributed\ConnectionRepository.py", line 620, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 1224, in handleDatagram
    self.handler(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1152, in handleMessageType
    self.handleGenerateWithRequired(di)
  File "otp\distributed\OTPClientRepository.py", line 1282, in handleGenerateWithRequired
    return self.__doGenerate(doId, parentId, zoneId, classId, di, other)
  File "otp\distributed\OTPClientRepository.py", line 1346, in __doGenerate
    self.generateWithRequiredFields(dclass, doId, di, parentId, zoneId)
  File "C:\Panda3D-1.10.0\direct\distributed\ClientRepositoryBase.py", line 254, in generateWithRequiredFields
    distObj = classDef(self)
  File "toontown\ai\NewsManager.py", line 16, in __init__
    base.localAvatar.inventory.setInvasionCreditMultiplier(1)
AttributeError: ToonBase instance has no attribute 'localAvatar'
Traceback (most recent call last):
  File "otp\otpbase\OTPBase.py", line 34, in run
    taskMgr.run()
  File "C:\Panda3D-1.10.0\direct\task\Task.py", line 508, in run
    self.step()
  File "C:\Panda3D-1.10.0\direct\task\Task.py", line 465, in step
    self.mgr.poll()
  File "C:\Panda3D-1.10.0\direct\showbase\ShowBase.py", line 1682, in updateManagers
    dt = globalClock.getDt()
  File "C:\Panda3D-1.10.0\direct\distributed\ConnectionRepository.py", line 613, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.10.0\direct\distributed\ConnectionRepository.py", line 620, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 1224, in handleDatagram
    self.handler(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1152, in handleMessageType
    self.handleGenerateWithRequired(di)
  File "otp\distributed\OTPClientRepository.py", line 1282, in handleGenerateWithRequired
    return self.__doGenerate(doId, parentId, zoneId, classId, di, other)
  File "otp\distributed\OTPClientRepository.py", line 1346, in __doGenerate
    self.generateWithRequiredFields(dclass, doId, di, parentId, zoneId)
  File "C:\Panda3D-1.10.0\direct\distributed\ClientRepositoryBase.py", line 254, in generateWithRequiredFields
    distObj = classDef(self)
  File "toontown\ai\NewsManager.py", line 16, in __init__
    base.localAvatar.inventory.setInvasionCreditMultiplier(1)
AttributeError: ToonBase instance has no attribute 'localAvatar'
